src/partials/section/schedule.html.eco

```diff
@@ -1,6 +1,4 @@
 <h2 class="section-title"><%= @labels['schedule'] %></h2>
-<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.</p>
```

Comment:
You can optionally add a small text explaining we'll have three days event.

Reply:
Yeah, but we already have this explanation on c4p section. I don't think so that we needed.